docs/atlascli/command/atlas-streams-instances-list.txt

```diff
@@ -0,0 +1,94 @@
+.. _atlas-streams-instances-list:
+
+============================
+atlas streams instances list
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+List all the Atlas Stream Processing instances for your project.
+
+To use this command, you must authenticate with a user account or an API key that has the Project Read Only role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas streams instances list [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for list
+   * - --limit
+     - int
+     - false
+     - Number of items per results page, up to a maximum of 500. If you have more than 500 results, specify the --page option to change the results page. This value defaults to 100.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --page
+     - int
+     - false
+     - Page number that specifies a page of results. This value defaults to 1.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   ID     NAME     CLOUD                               REGION
+   <Id>   <Name>   <DataProcessRegion.CloudProvider>   <DataProcessRegion.Region>
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # Return a JSON-formatted list of all Atlas Stream Processing instances for the project with ID 5e2211c17a3e5a48f5497de3:
+   atlas streams list --projectId 5e2211c17a3e5a48f5497de3 --output json
```

Comment:
atlas streams instance list -o json?

Reply:
Resolved in [82c2186](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/82c2186c49ac36d4c34bfb1057ac3be1bb66941d)